internal/authorizationmodel/read-from-input.go

```diff
@@ -54,7 +57,7 @@ func ReadFromFile(
 		*storeName = strings.TrimSuffix(path.Base(fileName), filepath.Ext(fileName))
 	}
 
-	return nil
+	return fileName, nil
```

Comment:
After playing with this further I think I might revert this change of returning the filename as it feels kind awkward. Will see what I can do to improve